spec/dummy/config/environments/test.rb

```diff
@@ -30,8 +30,8 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Store uploaded files on the local file system in a temporary directory
-  config.active_storage.service = :test
+  # # Store uploaded files on the local file system in a temporary directory
+  # config.active_storage.service = :test
```

Comment:
?

Reply:
We aren't using active_storage, so we don't need this configuration